### PR TITLE
lazycat 1.0.96 (new cask)

### PR DESCRIPTION
**Pull request by huhu415**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---
i do not know `Checked the cask was not` means, is this pr ok?

### CODE REVIEW

**Comment by huhu415** [comment]
what is wrong , what should I change ?

**Comment by SMillerDev** [comment]
Why did you call the PR `transmission 1.0.96 (new cask)` when you're not making a PR to add a `transmission` cask?

File: Casks/l/lazycat.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Documentation: https://docs.brew.sh/Cask-Cookbook
+#                https://docs.brew.sh/Adding-Software-to-Homebrew#cask-stanzas
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (SMillerDev) [inline Casks/l/lazycat.rb:3]: Please do this

File: Casks/l/lazycat.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+cask "lazycat" do
+  version "v1.0.96"
+
+  on_arm do
+    sha256 "8bfb3c6c64d62ee2fb91778e7db3cf8fb0ce90efa4f579aac10521c20ae9cb48"
+
+    url "https://dl.lazycat.cloud/client/desktop/stable/lzc-client-desktop_#{version}_arm64.dmg"
+  end
```
Comment (SMillerDev) [inline Casks/l/lazycat.rb:11]: This should use https://docs.brew.sh/Cask-Cookbook#handling-different-system-configurations

**Comment by huhu415** [comment]
I finally passed all the tests

File: Casks/l/lazycat.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+cask "lazycat" do
+  arch arm: "arm64", intel: "x64"
+
+  version "v1.0.96"
```
Comment (krehel) [inline Casks/l/lazycat.rb:4]:
```suggestion
  version "1.0.96"
```

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/l/lazycat.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  homepage "https://lazycat.cloud/"
+
+  livecheck do
+    url "https://dl.lazycat.cloud/client/desktop/lzc-client-desktop_#{arch}.dmg.metadata.json"
+    strategy :json do |json|
+      json["buildVersion"]
+    end
+  end
+
+  depends_on macos: ">= :catalina" # >=10.15
```
Comment (krehel) [inline Casks/l/lazycat.rb:20]:
```suggestion
  depends_on macos: ">= :catalina"
```

**Comment by huhu415** [comment]
I have a question for you 🙋, if the user `uninstalls` without executing `brew uninstall --zap firefox`, is there any way to remove those files afterwards?

**Comment by huhu415** [comment]
I finally passed all the tests again

File: Casks/l/lazycat.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
+cask "lazycat" do
+  arch arm: "arm64", intel: "x64"
+
+  version "1.0.96"
+  sha256  arm:   "8bfb3c6c64d62ee2fb91778e7db3cf8fb0ce90efa4f579aac10521c20ae9cb48",
+          intel: "3f00f3bbb8d86c86424e4ec2b9d2d0194ec0c63eceae300ed5b3bce1e9349a71"
+
+  url "https://dl.lazycat.cloud/client/desktop/stable/lzc-client-desktop_v#{version}_#{arch}.dmg"
+  name "LazyCat"
+  desc "懒猫微服客户端"
```
Comment (bevanjkay) [inline Casks/l/lazycat.rb:10]: Can the `desc` please be changed to an English version?

**Comment by huhu415** [comment]
Okay, I've changed it to English.

**Review by bevanjkay — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Review by krehel — APPROVED** [review]

**Pull request merged**